### PR TITLE
Prepare file structure

### PR DESCRIPTION
**Pull request by Ewelina-EN**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
```
Comment (agatakozinska) [inline public/index.html:12]: don't leave commented stuff in your code

File: src/data.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+  "destinations": [
+    {
+      "name": "Moon",
+      "images": {
+        "png": "./assets/destination/image-moon.png",
+        "webp": "./assets/destination/image-moon.webp"
```
Comment (agatakozinska) [inline src/data.json:7]: do you need webp ?

File: src/data.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      "role": "Flight Engineer",
+      "bio": "Anousheh Ansari is an Iranian American engineer and co-founder of Prodea Systems. Ansari was the fourth self-funded space tourist, the first self-funded woman to fly to the ISS, and the first Iranian in space."
+    }
+  ],
+  "technology": [
+    {
+      "name": "Launch vehicle",
+      "images": {
+        "portrait": "./assets/technology/image-launch-vehicle-portrait.jpg",
+        "landscape": "./assets/technology/image-launch-vehicle-landscape.jpg"
```
Comment (agatakozinska) [inline src/data.json:87]: do you need both images? can't you reach the result using css ? (comment applies for the rest of json)

File: src/index.js [hunk 4]
```diff
@@ -1,10 +1,9 @@
...
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const root = ReactDOM.createRoot(document.getElementById("root"));
```
Comment (agatakozinska) [inline src/index.js:6]: please configure eslint & prettier first, I think it should be single quote

File: src/App.js [hunk 5]
```diff
@@ -1,25 +1,5 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <div className="App"></div>;
```
Comment (agatakozinska) [inline src/App.js:2]: is it placeholder just for now, right?

**Review by agatakozinska — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 6]
```diff
@@ -1,25 +1,5 @@
-import logo from './logo.svg';
-import './App.css';
-
 function App() {
```
Comment (agatakozinska) [inline src/App.js:1]: maybe you could use a new syntax - arrow function?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions 
https://medium.com/codingthesmartway-com-blog/modern-react-from-the-beginning-ep5-arrow-functions-and-object-destructuring-468e5ceb6c77

File: src/data.json (hunk 2 above)
Comment (Ewelina-EN) [reply src/data.json]: no, I removed it thanks

File: src/data.json (hunk 3 above)
Comment (Ewelina-EN) [reply src/data.json:79]: now I will leave it, but I try to do it with css

File: src/index.js (hunk 4 above)
Comment (Ewelina-EN) [reply src/index.js:6]: configuration done. after merge a branch, there should be a single quote

File: src/App.js (hunk 5 above)
Comment (Ewelina-EN) [reply src/App.js:2]: yes :)

File: src/App.js (hunk 6 above)
Comment (Ewelina-EN) [reply src/App.js]: I changed that. thanks for link

**Review by agatakozinska — APPROVED** [review]

**Pull request merged**